client/me/purchases/manage-purchase/index.jsx

```diff
@@ -214,7 +206,7 @@ class ManagePurchase extends Component {
 		};
 
 		let text,
-			link = cancelPurchase( this.props.selectedSite.slug, id );
+			link = cancelPurchase( this.props.siteSlug, id );
```

Comment:
This link necessitates modification of `CancelPurchase` to prepare it to work with no `selectedSite`. With no changes, this would redirect away from cancel purchase route.